src/reference/asciidoc/mail.adoc

```diff
@@ -96,6 +96,33 @@ header of `application/octet-stream`.
 When you do not provide a header mapper, the message payload is the `MimeMessage` presented by `javax.mail`.
 The framework provides a `MailToStringTransformer` which can be used to convert the message using a simple strategy
 to convert the mail contents to a String.
+This is also available using the XML namespace:
+
+[source, xml]
+----
+<int-mail:mail-to-string-transformer ... >
+----
+
+and with Java configuration:
+
+[source, java]
+----
+@Bean
+@Transformer(inputChannel="...", outputChannel="...")
+public Transformer transformer() {
+    return new MailToStringTransformer();
+}
+----
+
+and with the Java DSL:
+
+[source, java]
+----
+   ...
+   .transform(new MailToStringTransformer())
```

Comment:
H-m-m. OK. 
Good catch!

Feel free to raise a GH issue against DSL and even consider a Contribution! 

Anyway will fix with the `Transformers` to promote it a bit. :wink: